packages/react/src/table/TableCell.tsx

```diff
@@ -11,6 +11,7 @@ export type TableCellProps = PropsWithChildren<{
 	display?: ResponsiveProp<'none' | 'table-cell'>;
 	/** Sets the font weight of the cell. */
 	fontWeight?: FontWeight;
+	id?: string;
```

Comment:
JSDoc?

Reply:
Will add `/** Defines an identifier (ID) which must be unique. */` as that seems to be the standard